pom.xml

```diff
@@ -64,7 +64,7 @@
     <surefire.version>2.19.1</surefire.version>
     <testng.version>6.9.10</testng.version>
     <guava.version>18.0</guava.version>
-    <cxf.version>3.1.10</cxf.version>
+    <cxf.version>3.2.7</cxf.version>
```

Comment:
With versions of `cxf` after 3.2.2, I get the following error in the tests:

```
javax.xml.ws.soap.SOAPFaultException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse

	at org.apache.cxf.jaxws.JaxWsClientProxy.mapException(JaxWsClientProxy.java:195)
	at org.apache.cxf.jaxws.JaxWsClientProxy.invoke(JaxWsClientProxy.java:145)
	at com.sun.proxy.$Proxy49.delete(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at io.cloudsoft.winrm4j.client.RetryingProxyHandler.invokeWithRetry(RetryingProxyHandler.java:44)
	at io.cloudsoft.winrm4j.client.RetryingProxyHandler.invoke(RetryingProxyHandler.java:34)
	at com.sun.proxy.$Proxy50.delete(Unknown Source)
	at io.cloudsoft.winrm4j.client.ShellCommand.close(ShellCommand.java:209)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.doRequest(WinRmClientRecordedTest.java:85)
	at io.cloudsoft.winrm4j.client.WinRmClientRecordedTest.testRecording(WinRmClientRecordedTest.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.cxf.endpoint.ClientImpl$IllegalEmptyResponseException: Response message did not contain proper response data. Expected: {http://schemas.dmtf.org/wbem/wsman/1/wsman.xsd}DeleteResponse
	at org.apache.cxf.endpoint.ClientImpl.processResult(ClientImpl.java:663)
	at org.apache.cxf.endpoint.ClientImpl.doInvoke(ClientImpl.java:540)
	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:440)
	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:355)
	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:313)
	at org.apache.cxf.frontend.ClientProxy.invokeSync(ClientProxy.java:96)
	at org.apache.cxf.jaxws.JaxWsClientProxy.invoke(JaxWsClientProxy.java:140)
	... 23 more
```

Reply:
@nakomis I think this commit (https://github.com/apache/cxf/commit/1396f046333108ec0e7819bf4de154505ff4564c) in `cxf` is what is causing our problem. It was added as part of the cxf-3.2.3 release. (`git bisect` dug it out)